src/java.base/share/classes/java/util/ResourceBundle.java

```diff
@@ -509,7 +509,7 @@ public ResourceBundle() {
      * Calling this method is equivalent to calling
      * {@snippet lang=java :
      *     // @link substring="getObject" target="#getObject(java.lang.String)"
```

Comment:
Can you do a drive-by update for the comment in the snippet here,

```suggestion
     *     // @link substring="getObject" target="#getObject(java.lang.String)" :
```

and also for the one in `getStringArray(String key)`